src/frontend/web/src/ratingpage/ArrowIcon.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
Not related to this file - the name of the directory is `ratingPage` as referred from `App.js` but here it remained `ratingpage`. I think it is best to call it `RatingPage` and change it everywhere.